docs/concepts/Override/README.md

```diff
@@ -0,0 +1,275 @@
+# Override
+
+## Overview
+The `ClusterResourceOverride` and `ResourceOverride` provides a way to customize resource configurations before they are propagated 
+to the target cluster by the `ClusterResourcePlacement`.
+
+## Difference Between `ClusterResourceOverride` And `ResourceOverride`
+
+`ClusterResourceOverride` represents the cluster-wide policy that overrides the cluster scoped resources to one or more
+clusters while `ResourceOverride` will apply to resources in the same namespace as the namespace-wide policy.
+
+> **Note:** If a namespace is selected by the `ClusterResourceOverride`, ALL the resources under the namespace are selected
+automatically.
+
+If the resource is selected by both `ClusterResourceOverride` and `ResourceOverride`, the `ResourceOverride` will win
+when resolving the conflicts.
+
+## When To Use Override
+Overrides is useful when you want to customize the resources before they are propagated from the hub cluster to the target clusters.
+Some example use cases are:
+- As a platform operator, I want to propagate a clusterRoleBinding to cluster-us-east and cluster-us-west and would like to
+grant the same role to different groups in each cluster.
+- As a platform operator, I want to propagate a clusterRole to cluster-staging and cluster-production and would like to
+grant more permissions to the cluster-staging cluster than the cluster-production cluster.
+- As a platform operator, I want to propagate a namespace to all the clusters and would like to customize the labels for
+each cluster.
+- As an application developer, I would like to propagate a deployment to cluster-staging and cluster-production and would
+like to always use the latest image in the staging cluster and a specific image in the production cluster.
+- As an application developer, I would like to propagate a deployment to all the clusters and would like to use different
+commands for my container in different regions.
+
+## Limits
+- Each resource can be only selected by one override simultaneously. In the case of namespace scoped resources, up to two
+overrides will be allowed, considering the potential selection through both `ClusterResourceOverride` (select its namespace) 
+and `ResourceOverride`.
+- At most 100 `ClusterResourceOverride` can be created.
+- At most 100 `ResourceOverride` can be created.
+
+## Resource Selector
+`ClusterResourceSelector` of `ClusterResourceOverride` selects which cluster-scoped resources need to be overridden before

```

Comment:
Add how `ResourceSelector` used for `ResourceOverride` to select namespace-scoped resources.